ietf/doc/models.py

```diff
@@ -1238,8 +1227,6 @@ def __str__(self):
 
     def save(self, *args, **kwargs):
```

Comment:
Get rid of this now redundant method?

Reply:
I don't think this is redundant? It's possible (and I think necessary in some places) to save a docevent without saving a change to the referenced Document, and we have a lot of construct that creates and saves docevents before calling save_with_history on Document. Do I misunderstand what you're pointing to as redundant?

Reply:
Or is the point that we should just rely on models.Model.save()?